modules/mono/editor/GodotTools/GodotTools.ProjectEditor/IdentifierUtils.cs

```diff
@@ -33,22 +33,25 @@ private static void SkipInvalidCharacters(string source, int startIndex, StringB
 
                 switch (char.GetUnicodeCategory(@char))
                 {
-                    case UnicodeCategory.UppercaseLetter:
-                    case UnicodeCategory.LowercaseLetter:
-                    case UnicodeCategory.TitlecaseLetter:
-                    case UnicodeCategory.ModifierLetter:
-                    case UnicodeCategory.LetterNumber:
-                    case UnicodeCategory.OtherLetter:
+                    case UnicodeCategory.UppercaseLetter: break;
+                    case UnicodeCategory.LowercaseLetter: break;
+                    case UnicodeCategory.TitlecaseLetter: break;
+                    case UnicodeCategory.ModifierLetter: break;
+                    case UnicodeCategory.LetterNumber: break;
+                    case UnicodeCategory.OtherLetter: break;
                         outputBuilder.Append(@char);
```

Comment:
This is also completely wrong, they are supposed to be fallthrough so that all cases will run `outputBuilder.Append(@char);`.